The current rocketmq logger incompatible with third-party libraries

```
java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
	at io.openmessaging.storage.dledger.DLedgerServer.<clinit>(DLedgerServer.java:69)
	at org.apache.rocketmq.controller.impl.DLedgerController.<init>(DLedgerController.java:113)
	at org.apache.rocketmq.controller.ControllerManager.initialize(ControllerManager.java:98)
	at org.apache.rocketmq.namesrv.NamesrvStartup.start(NamesrvStartup.java:201)
	at org.apache.rocketmq.namesrv.NamesrvStartup.createAndStartControllerManager(NamesrvStartup.java:180)
	at org.apache.rocketmq.namesrv.NamesrvStartup.controllerManagerMain(NamesrvStartup.java:73)
	at org.apache.rocketmq.namesrv.NamesrvStartup.main(NamesrvStartup.java:54)
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
```
@aaron-ai

--- a/controller/pom.xml
+++ b/controller/pom.xml
@@ -42,6 +42,14 @@
             </exclusions>
         </dependency>
         <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+        </dependency>
+        <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-client</artifactId>
             <scope>test</scope>
@@ -49,11 +57,6 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-srvutil</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <scope>test</scope>
         </dependency>
     </dependencies>
 </project>

